Error while checking build/O2/fullCI_slc9 for 1df1b850cdd6b5685b0f664fc29869649d140184 at 2025-10-11 16:09:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/14730-slc9_x86-64/0/Detectors/TOF/workflow/src/make-parameter-collection.cxx:66:3: error: 'ParamExample::~ParamExample()' marked 'override', but does not override
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/14730/1df1b850cdd6b5685b0f664fc29869649d140184/build_O2_fullCI_slc9/pretty.html).

--- a/Detectors/TOF/workflow/src/make-parameter-collection.cxx
+++ b/Detectors/TOF/workflow/src/make-parameter-collection.cxx
@@ -63,7 +63,7 @@ class ParamExample : public Parameters<5>
  public:
   ParamExample() : Parameters(std::array<std::string, 5>{"p0", "p1", "p2", "p3", "p4"},
                               "ParamExample") { setParameters(std::array<paramvar_t, 5>{0, 1, 2, 3, 4}); }; // Default constructor with default parameters
-  ~ParamExample() override = default;
+  ~ParamExample() = default;
 };
 
 int main(int argc, char* argv[])

